tests/acceptance/features/apiSpaces/quota.feature

```diff
@@ -0,0 +1,75 @@
+    And user "Alice" lists all available spaces via the GraphApi
+    And user "Alice" uploads a file inside space "<spaceName>" with content "<fileContent>" to "test.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    When user "Alice" lists all available spaces via the GraphApi
+    Then the json responded should contain a space "<spaceName>" with these key and value pairs:
+      | key              | value       |
+      | name             | <spaceName> |
+      | quota@@@state    | <state>     |
+      | quota@@@total    | <total>     |
+      | quota@@@remaining| <remaining> |
+      | quota@@@used     | <used>      |
+    Examples:
+      | spaceName | fileContent                                                                                           | state    | total | remaining | used |
+      | Quota1%   | 1                                                                                                    | normal   | 100   | 99        | 1    |
+      | Quota75%  | 123456789 123456789 123456789 123456789 123456789 123456789 123456789 12345                          | normal   | 100   | 25        | 75   |
+      | Quota76%  | 123456789 123456789 123456789 123456789 123456789 123456789 123456789 123456                         | nearing  | 100   | 24        | 76   |
+      | Quota90%  | 123456789 123456789 123456789 123456789 123456789 123456789 123456789 123456789 1234567890           | nearing  | 100   | 10        | 90   |
+      | Quota91%  | 123456789 123456789 123456789 123456789 123456789 123456789 123456789 123456789 123456789 1          | critical | 100   | 9         | 91   |
+      | Quota99%  | 123456789 123456789 123456789 123456789 123456789 123456789 123456789 123456789 123456789 123456789  | critical | 100   | 1         | 99   |
+      | Quota100% | 123456789 123456789 123456789 123456789 123456789 123456789 123456789 123456789 123456789 1234567890 | exceeded | 100   | 0         | 100  |
+
+  
+  Scenario: uploading a file with an insufficient quota
+    When user "Alice" creates a space "Project Alfa" of type "project" with quota "10" using the GraphApi
+    And user "Alice" lists all available spaces via the GraphApi
+    And user "Alice" uploads a file inside space "Project Alfa" with content "More than 10 bytes" to "test.txt" using the WebDAV API
+    Then the HTTP status code should be "507"
+
+
+  Scenario: creating a folder with an insufficient quota
+    When user "Alice" creates a space "Project Beta" of type "project" with quota "7" using the GraphApi
+    And user "Alice" lists all available spaces via the GraphApi
+    And user "Alice" uploads a file inside space "Project Alfa" with content "7 bytes" to "test.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    When user "Alice" creates a folder "NewFolder" in space "Project Beta" using the WebDav Api
+    Then the HTTP status code should be "201" 
```

Comment:
I have found that, as well as checking the HTTP status code, it is good to check that the resource (folder) exists.
```
And the folder "NewFolder" exists in space "Project Beta"
```
(whatever the words are for that step)

Because there have been bugs where the API response returns a "success" 2xx code, but the server-under-test lied, and did not actually create the resource.